code/api/src/migrations/10-shipment-returned.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
I separated out the migrations because I forgot about the returned column

Reply:
No worries this looks good!